src/scoring.jl

```diff
@@ -267,6 +267,8 @@ function ck_workers_worktime(workers, shifts, workers_info)::ScoringResult
     max_overtime = num_weeks * MAX_OVERTIME
     max_undertime = num_weeks * MAX_UNDERTIME
 
+    holidays = length(filter((x) -> x % 7 != 6, get(month_info, "holidays", Int[])))
```

Comment:
Fajny ten get. 7 to rozumiem, że ilość dni w tygodniu, miałem na to jakąś stałą, możesz ją przy okazji ujdenilicić z ilością dni pracujących, w sensie nazwą. 

btw. 6 to sobota ;)

Reply:
Fajnie jakbyś to jakoś opisał, może być komentarz, bo troche nie wiadomo co się dzieje

Reply:
a i tego nawiasu okrągłego nie musi być `(x)`

Reply:
No ok, to zaraz podmienię na stałą, niedzielę też mogę do stałych podrzucić, żeby było czytelniej xD.
Czy tydzień u nas zaczyna się od niedzieli?

Reply:
Co do komentarzy miałem propozycję właśnie co do nich i algorytmu. Ogólnie myślę, że można by było trochę pokomentować cały solver w jakiś bardziej złożonych miejscach, co by ułatwiło szukanie w nim potrzebnych rzeczy, plus gdyby kiedyś przyszły kolejne osoby, miałyby niższy próg wejścia

Reply:
Zgadzam się co do komentarzy! 

Tydzień zaczyna się od poniedziałku, ale jak chcesz wykluczyć niedzielę (dzień 7), to reszta musi być różna od 0.
Jest trochę inaczej, ponieważ w Julii numerujemy od 1.

Reply:
Ah, dobra, zapomniałem o tym że tu się indeksuje od 1